test/tests/materials/active_material_props/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [check_active_properties]
+    issues = '#26696'
+    design = 'syntax/Materials/index.md'
+    requirement = 'The system shall provide a function for materials to check whether a declared material property is active on current element.'
+    type = 'Exodiff'
```

Comment:
why an exodiff? The properties are outputting to console?

Reply:
Not only that. I added a test aux kernel and used it here in this test.

Reply:
It seems to be testing a variety of behavior with bcs, kernels and auxkernels.
you ll need to document the expected result next to every output variable. 

and why can properties not be active when consumed by an auxkernel?

Reply:
The aux kernel `CheckActiveMatProp` is not calling `getMaterialProperty` but directly checking whether a material property is active through `MaterialData`.

Reply:
I will add some inline documents in the input.

Reply:
@GiudGiud I added tables in the input to show where and when the properties are active based on kernels,bcs,postprocessors,auxkernels.